catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -56,7 +56,7 @@ public CreateRequest processPreCreate(CreateRequest input) throws StopProcessing
             input.getMetacards()
                     .forEach(metacard -> {
                         metacard.setAttribute(new AttributeImpl(Metacard.POINT_OF_CONTACT,
-                                SubjectUtils.getName((ddf.security.Subject) subjectName)));
+                                SubjectUtils.getEmailAddress((ddf.security.Subject) subjectName)));
```

Comment:
❓ Also, what should happen if the email address is ever `null`? Not sure if it's possible, but something to consider.